openupgradelib/openupgrade.py

```diff
@@ -811,8 +811,8 @@ def map_values(
     mapped
     :param target_column: the database column, or model field (if 'write' is \
     'orm') that the new values are written to
-    :para mapping: list of tuples [(old value, new value)]
-    Old value True represents "is set", False "is not set".
+    :param mapping: list of tuples [(old value, new value)]
+        Old value True represents "is set", False "is not set".
```

Comment:
I have done something similar in the doc of OdooRPC: https://github.com/osiell/odoorpc/blob/master/odoorpc/models.py#L232
The line above has a `\`, that's why the indentation is different I think.